lsp/eslint.lua

```diff
@@ -0,0 +1,182 @@
+local util = require 'lspconfig.util'
+local lsp = vim.lsp
+
+-- @brief
+--
+-- https://github.com/hrsh7th/vscode-langservers-extracted
+--
+-- `vscode-eslint-language-server` is a linting engine for JavaScript / Typescript.
+-- It can be installed via `npm`:
+--
+-- ```sh
+-- npm i -g vscode-langservers-extracted
+-- ```
+--
+-- `vscode-eslint-language-server` provides an `EslintFixAll` command that can be used to format a document on save:
+-- ```lua
+-- vim.lsp.config('eslint', {
+--   --- ...
+--   on_attach = function(client, bufnr)
+--     vim.api.nvim_create_autocmd("BufWritePre", {
+--       buffer = bufnr,
+--       command = "EslintFixAll",
+--     })
+--   end,
+-- })
+-- ```
+--
+-- See [vscode-eslint](https://github.com/microsoft/vscode-eslint/blob/55871979d7af184bf09af491b6ea35ebd56822cf/server/src/eslintServer.ts#L216-L229) for configuration options.
+--
+-- Messages handled in lspconfig: `eslint/openDoc`, `eslint/confirmESLintExecution`, `eslint/probeFailed`, `eslint/noLibrary`
+--
+-- Additional messages you can handle: `eslint/noConfig`
+
+return {
+  cmd = { 'vscode-eslint-language-server', '--stdio' },
+  filetypes = {
+    'javascript',
+    'javascriptreact',
+    'javascript.jsx',
+    'typescript',
+    'typescriptreact',
+    'typescript.tsx',
+    'vue',
+    'svelte',
+    'astro',
+  },
+  on_init = function(client)
+    vim.api.nvim_create_user_command('EslintFixAll', function()
```

Comment:
commands must be prefixed with "Lsp"
```suggestion
    vim.api.nvim_create_user_command('LspEslintFixAll', function()
```